datadog_checks_base/datadog_checks/base/utils/warnings_util.py

```diff
@@ -0,0 +1,36 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import warnings
+from contextlib import contextmanager
+
+from six import PY2
+
+
+def _simplefilter_py2(action, category=Warning, lineno=0, append=0):
+    """ Add remove logic for py2 to avoid warnings.filters to growth
+        indefinitely if simplefilter is called multiple times """
```

Comment:
Can you format the docstring a bit more nicely.

Reply:
fixed